migrations/capcons/migration_test.go

```diff
@@ -1818,6 +1921,11 @@ func TestLinkMigration(t *testing.T) {
 					capabilityID: 2,
 				},
 			},
+			expectedEvents: []string{
+				`flow.AccountCapabilityControllerIssued(id: 1, address: 0x0000000000000001, type: Type<auth(Capabilities,Contracts,Inbox,Keys,Storage)&Account>())`,
+				// TODO: fix type
+				`flow.AccountCapabilityControllerIssued(id: 2, address: 0x0000000000000001, type: Type<&auth(Capabilities,Contracts,Inbox,Keys,Storage)&Account>())`,
```

Comment:
It looks like there's a bug in the link migration and the test cases for it. I'll fix this in a separate PR